api/external/events.html

```diff
@@ -0,0 +1,530 @@
+                                    <col class="field-name" />
+                                    <col class="field-body" />
+                                    <tbody valign="top">
+                                    <tr class="field-even field">
+                                        <th class="field-name">Raises:</th>
+                                        <td class="field-body">
+                                            <tt class="docutils literal">
+                                                <span class="pre">Http404</span>
+                                            </tt>
+                                            if no event type with the specified ID is found.
+                                        </td>
+                                    </tr>
+                                    </tbody>
+                                </table>
+                            </dd>
+                        </dl>
+                        <p>With these endpoints, you can create member events and view information about specific member's events.</p>
+                        <dl class="post">
+                            <dt id="post--#account_id-member_event">
+                                <tt class="descname">POST </tt>
+                                <tt class="descname">/#account_id/events/members</tt>
+                                <a class="headerlink" href="#post--#account_id-member_event" title="Permalink to this definition">&para;</a>
+                            </dt>
+                            <dd>
+                                <p>Create up to 10 member events.</p>
+                                <table class="docutils field-list" frame="void" rules="none">
+                                    <col class="field-name" />
+                                    <col class="field-body" />
+                                    <tbody valign="top">
+                                    <tr class="field-odd field"><th class="field-name">Parameters:</th>
+                                        <td class="field-body">
+                                            <ul class="first simple">
+                                                <li><strong>events</strong> (<em>array of dictionaries</em>) &#8211; Required. A list of member event dictionaries.
+                                                    <ul class="first simple">
+                                                        <li><strong>email</strong> (<em>string</em>) &#8211; The member email.</li>
+                                                        <li><strong>member_id</strong> (<em>string</em>) &#8211; The member ID.</li>
+                                                        <li><strong>event_type_id</strong> (<em>string</em>) &#8211; The event type ID.</li>
+                                                        <li><strong>event_type</strong> (<em>string</em>) &#8211; The event type display name.</li>
```

Comment:
This value is an `event_type`, not a display name

```suggestion
                                                        <li><strong>event_type</strong> (<em>string</em>) &#8211; An existing defined event_type.</li>
```